ubuntu20.04/val-verde-platform-sdk-package-build

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /sources/configure-deb-control
```

Comment:
This should be after the set -e.

Reply:
removed this file to address deb generation as a separate PR